generators/server/templates/src/test/java/package/web/rest/PublicUserResourceIT.java.ejs

```diff
@@ -113,6 +113,10 @@ class PublicUserResourceIT <% if (databaseType === 'cassandra') { %>extends Abst
 
     private static final String DEFAULT_LOGIN = "johndoe";
 
+    private static final String DEFAULT_FIRSTNAME = "john";
+
+    private static final String DEFAULT_LASTNAME = "doe";
+
```

Comment:
```suggestion
<%_ for (field of _.sortedUniqBy(user.otherRelationships.map(relationship => relationship.relatedField), field => field.fieldName)) { _%>
    private static final String DEFAULT_<%= field.fieldName.toUpperCase %> = "<%= field.fieldName %>";

<%_ } _%>
```

Reply:
this code this code's making IT crash, by generate wrong constatns, for firstName should be john and for lastName should be doe